docs/guides/cloud/analytics/enterprise-reporting.mdx

```diff
@@ -52,24 +54,59 @@ on this tab correlates to what Cypress Cloud [counts as a test result](/faq/ques
 
 ### Test Results
 
-How many of your tests are passing or failing within builds, specs, and individual test runs? The `Test Results` tab
+How many of your runs, specs, or individual tests are passing or failing? The **Test Results** tab
 of Enterprise Reporting allows you to review the status rates of your tests from all three levels of granularity.
 
 <DocsImage
   src="/img/guides/cloud/analytics/enterprise-reporting-test-results.png"
   alt="Cloud Enterprise Reporting test results report"
 />
 
-### Test Suite
+### Test Setup
+
+The **Test Setup** tab on Enterprise Reporting contains metrics that show how your runs are configured and maintained.
+What is your proportion of end-to-end tests to [component tests](/guides/component-testing/overview)? How has the size of your
```

Comment:
Would these questions be better as a bulleted list?

Reply:
maybe - i'll mock one up and take a quick look